src/Prism.Core/Common/ParametersBase.cs

```diff
@@ -74,32 +90,72 @@ protected ParametersBase(string query)
             }
         }
 
+        /// <summary>
+        /// The count, or number, of parameters in collection
+        /// </summary>
         public int Count => _entries.Count;
 
+        /// <summary>
+        /// Returns an IEnumerable of the Keys in the collection
+        /// </summary>
         public IEnumerable<string> Keys => 
             _entries.Select(x => x.Key);
 
+        /// <summary>
+        /// Adds the <paramref name="key" /> and <paramref name="value" /> to the KeyValuePair<string,object> collection
+        /// </summary>
+        /// <param name="key">The key to reference this value in the KeyValuePair<string, object></param>
+        /// <param name="value">The value of the parameter to store</param>
         public void Add(string key, object value) =>
             _entries.Add(new KeyValuePair<string, object>(key, value));
 
+        /// <summary>
+        /// Checks collection for presense of key
+        /// </summary>
+        /// <param name="key">The key to check in the collection</param>
+        /// <returns>True if key exists; else returns false.</returns>
         public bool ContainsKey(string key) =>
             _entries.ContainsKey(key);
 
+        /// <summary>
+        /// Gets an enumerator for the KeyValuePairs in parameter collection
+        /// </summary>
+        /// <returns>Enumerator</returns>
         public IEnumerator<KeyValuePair<string, object>> GetEnumerator() =>
             _entries.GetEnumerator();
-
+        /// <summary>
+        /// Returns the value of the member referenced by key
+        /// </summary>
+        /// <typeparam name="T">The type of object to be returned</typeparam>
+        /// <param name="key">The key for the value to be returned</param>
+        /// <returns>Returns a matching parameter of <typeparam name="T" /> if one exists in the Collection</returns>
```

Comment:
Use `<see cref="T" />` not `<typeparam name="T" />`

Reply:
I have incorporated the requested change